docs/source/installation/developer-anaconda.rst

```diff
@@ -0,0 +1,43 @@
+Developer installation (Anaconda Cloud)
+=======================================
+
+Developer installations are useful for those who want to customise the *labscript suite*.
+
+.. note:: You need not fork, clone, and install editable versions of all *labscript suite* repositories to customise your installation and/or contribute changes back to the base repositories.
+    For example, if you only want to develop custom labscript device drivers, you might only fork and clone the labscript-devices repository.
+    Moreover, there is now an option to write and use custom labscript device drivers outside of the labscript-devices installation directory.
+
+
+In this example, we will use an existing conda environment named `py38`.
+Skip the first line/step if continuing on from the instructions to :ref:`set up this environment <installation/setting-up-an-environment:Anaconda Python>`.
+
+
+Quick start
+-----------
+
+.. note:: After the first line, the current directory is ommited from the command prompt
+    for brevity.
+
+.. code-block:: console
+
+    (base) C:\Users\wkheisenberg> mkdir labscript-suite
```

Comment:
This folder is not consistent with the other documentation that uses `C:\labscript-suite`

Reply:
This might be anticipating a prospective change being discussed in #43, whereby we recommend using the profile directory for a developer installation.

Reply:
Oh, I thought that was just going to be for the .venv, not the editable installs too. Did I misinterpret?

Reply:
I understood it to mean for both, which I think is a good idea.

Reply:
A subdirectory of the profile directory would also be fine, but a different but inevitably similarly-named directory outside it is unappealing to me.